bmq-sdk/src/test/java/com/bloomberg/bmq/it/SessionIT.java

```diff
@@ -3468,16 +3468,19 @@ public void testQueueFlushDown() throws Exception {
         //   broker.kill();
         //   session.waitConnectionLostEvent(); // There will be no ACK messages sent
 
+        // Create, but don't start broker yet
+        // Use broker in the second part of test where PUTs are sent
+        // and corresponding ACKs are expected
+        BmqBroker broker = BmqBroker.createStoppedBroker();
+
         // Use simulator in the first part of test where PUTs are sent
         // and there should be no ACKs
         BmqBrokerSimulator simulator =
-                new BmqBrokerSimulator(BmqBrokerSimulator.Mode.BMQ_MANUAL_MODE);
+                new BmqBrokerSimulator(
+                        broker.sessionOptions().brokerUri().getPort(),
```

Comment:
broker, simulator, session must have the same port
Actually I am surprised that this test passed before

Reply:
Indeed ... ))